packages/flutter_test/lib/src/binding.dart

```diff
@@ -125,6 +125,15 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   @protected
   bool get disableShadows => false;
 
+  /// Abstract method, implemented in [AutomatedTestWidgetsFlutterBinding] only.
```

Comment:
Can you replace this paragraph with `Increase the timeout for the current test by the given duration.`? The doc comment shouldn't explain how something is implemented, just what it is used for.